docs/configuration/protocols/bgp.rst

```diff
@@ -530,7 +530,7 @@ Redistribution Configuration
 General Configuration
 ---------------------
 
-Common parametrs
+Common parameters
 ^^^^^^^^^^^^^^^^
```

Comment:
to prevent build warnings, please fill the title underline. The rest is good, thank you.

Reply:
I take it you mean, just do like:

Common parameters
---------------------

?

Reply:
yes just a single ^ character under the "s" from parameters, sphinx will raise a build warnings when the underline from the title have not the correct length.

Reply:
There we go sir, I see what you meant. That was totally my miss. Thank you for catching that.